api/v1alpha2/awscluster_conversion.go

```diff
@@ -187,3 +190,8 @@ func Convert_v1alpha3_AWSLoadBalancerSpec_To_v1alpha2_AWSLoadBalancerSpec(in *in
 func Convert_v1alpha3_ClassicELBAttributes_To_v1alpha2_ClassicELBAttributes(in *infrav1alpha3.ClassicELBAttributes, out *ClassicELBAttributes, s apiconversion.Scope) error { //nolint
 	return autoConvert_v1alpha3_ClassicELBAttributes_To_v1alpha2_ClassicELBAttributes(in, out, s)
 }
+
+//Convert_v1alpha3_NetworkSpec_To_v1alpha2_NetworkSpec
+func Convert_v1alpha3_NetworkSpec_To_v1alpha2_NetworkSpec(in *infrav1alpha3.NetworkSpec, out *NetworkSpec, s apiconversion.Scope) error {
```

Comment:
I don't think this is needed here, as it's part of the autogenerated bits?

Reply:
The autogenerated code was [removing](https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1747/files#diff-94d92e5ec55d2db907b66f93901cc2e2L212-L216) the Convert function so added it here since I saw some other fields here as well.